### ISSUE TITLE
It should be possible to move the editor

### ISSUE THREAD

**Issue by erikvold** [brianpeiris/RiftSketch#48]
It would be nice if I could move/position the editor somehow.

**Comment by brianpeiris** [comment]
Yup, I lost this feature after some refactoring. One workaround, assuming you're using it in VR mode, is that you can re-center your view if you hit `Ctrl/Alt+Shift+z`. Not quite the same, but it might help.
